app/services/mixpanel_service.rb

```diff
@@ -38,6 +38,11 @@ def initialize
   # @param [Hash] data: (optional, defaults to {}) data to be sent to mixpanel
   #
   def run(distinct_id:, event_name:, data: {})
+    unless Rails.env.production?
+      Rails.logger.error "You are currently on non-prod environment. If you want to track mixpanel events in non-prod environments, please uncomment this check in Mixpanel Service"
```

Comment:
is this going to clutter our sentry/datadog logs for non-prod environments? I don't know if we need to log an error for every event we don't track.